templates/react/utils/itemToLinks.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {API_PATH} from '../api/_entrypoint';
+
+export default function(items) {
+  return Array.isArray(items)
+    ? items.map(item => createLink(item))
+    : createLink(items);
+}
+
+function createLink(item) {
+  if (typeof(item) === 'string' && item.includes(API_PATH) > 0) {
```

Comment:
Looks suspicious, by default API Platform generates URLs that don't contain the domain.